From 195b8165ac35dfe9f5ba14e1a69bb88b7686c5a9 Mon Sep 17 00:00:00 2001
From: "David S. Miller" <davem@davemloft.net>
Date: Wed, 5 Feb 2014 22:13:13 -0800
Subject: [PATCH] Fix tst-sscanf and tst-swscanf on 64-bit.

	* stdio-common/tst-sscanf.c (main): Use 'long' for 'dummy' when
	processing int_tests.
---
 stdio-common/tst-sscanf.c | 3 ++-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/stdio-common/tst-sscanf.c b/stdio-common/tst-sscanf.c
index 1214c7d..9fef93a 100644
--- a/stdio-common/tst-sscanf.c
+++ b/stdio-common/tst-sscanf.c
@@ -186,7 +186,8 @@ main (void)
 
   for (i = 0; i < sizeof (int_tests) / sizeof (int_tests[0]); ++i)
     {
-      int dummy, ret;
+      long dummy;
+      int ret;
 
       if ((ret = SSCANF (int_tests[i].str, int_tests[i].fmt,
 			 &dummy)) != int_tests[i].retval)
-- 
1.9.0

